army.cpp

```diff
@@ -1050,8 +1071,13 @@ int Army::GetTargetNum(char const *special)
 		}
 	} else {
 		int i = getrandom(tars);
-		if (i<canfront) return i;
-		return i + canbehind - canfront;
+		if (canAttackBehind) {
+			return i;
+		}
+		else {
```

Comment:
can you please put else on a same line as `}`

Reply:
Yes, but I would need to rebase PR #93 , what I don't want now. I can fix it in #93